requirements.txt

```diff
@@ -1 +1,2 @@
 # List requirements here.
+pytest==7.2.1
```

Comment:
This is what we call a "development" requirement. It's listed here:
https://github.com/pcdshub/pcds-ci-test-repo-python/blob/master/dev-requirements.txt